DOCUMENTATION.md

```diff
@@ -742,6 +742,16 @@ If you leave out this argument, the client returns the most recent 250 notificat
 
 The client only returns notifications that are 7 days old or newer. If the notification specified in this argument is older than 7 days, the client returns an empty response.
 
+#### includeSpreadsheetUploads (optional)
+
+A boolean to set if you would like to include notifications that have been sent using a spreadsheet through the Notify interface.
```

Comment:
@quis I feel you may be able to offer me some suggestions on how to write all this better please :)